CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Changed
 - Updated event testing - read more [here](./docs/src/testing/testing-events.md) on how it now works and [here](./docs/src/appendix/cheatcodes/spy_events.md)
 about updated `spy_events` cheatcode
+- Updated `cheat_account_contract_address` docs [here](./docs/src/appendix/cheatcodes.md#account-contract-address) and added a `start_` prefix to the `..._global` invocation which works globally and indefinitely
```

Comment:
This is user-facing, we don't want the user to know that we've changed the docs
We also would like to link to the docs (you can kind of predict what page address there actually will be)

And also, it's in the wrong section, we don't modify the released changelogs 
This should be in `Unreleased` one

Reply:
Is [this](https://foundry-rs.github.io/starknet-foundry/appendix/cheatcodes/account_contract_address.html) doc page link expected here?